python/docstrings.json

```diff
@@ -27,25 +27,25 @@
         "signature": "get_dt() -> float",
         "doc": "Get the timestep size of the last completed step.\n\n:return: Timestep size in days.\n:type return: float"
     },
-    "getFluidStateVariables": {
-        "signature": "get_fluid_state_variables(name: str) -> NDArray[float]",
-        "doc": "Retrieve a fluid state variable for the simulation grid.\n\n:para name: The name of the variable. Valid names are 'pw' (pressure water), 'pg' (pressure gas), 'po' (pressure oil), 'rho_w' (density water), 'rho_g' (density gas), 'rho_o' (density oil)'Rs' (soultion gas-oil ratio), 'Rv' (volatile gas-oil ratio), 'Sw' (water saturation), 'Sg' (gas saturation), 'So' (oil saturation), and 'T' (temperature).\n:type name: str\n\n:return: An array of fluid state variables.\n:type return: NDArray[float]"
+    "getFluidStateVariable": {
```

Comment:
maybe split out the docstring fixes in a separate commit?

Reply:
Yes, I thought it would not be worth it but it might be a good ida to keep the commits and PRs strictly on topic. Btw, I think #5243 also includes these docstring fixes.

Reply:
it will make it easier in two years if we look back in history because something is broken with mpi. so i would really prefer commits to be kept on topic. and the two commits here squashed.

Reply:
Ok, noted! Should I revert docstring changes to `PyBlackOilSimulator.cpp` also?

Reply:
ah missed those. yes please keep out of the mpi commit. if you want you can include all those changes in this PR, that is no problem, just keep it in separate commits.

Reply:
perfect, thanks, sorry to be a PITA ;)